src/aspire/volume/volume.py

```diff
@@ -176,6 +179,46 @@ def _set_symmetry_group(self, value):
             )
         self._symmetry_group = value
 
+    def _symmetry_group_warning(self):
+        """
+        Warn when an operation has the potential to change the symmetry
+        of a volume or the alignment of symmetry axes.
+        """
+        msg = (
+            f"`symmetry_group` attribute is being set to `C1`. This operation may"
+            f" effect the symmetry (or symmetric alignment) of the volume. To reset the"
+            f" symmetry group run `self._set_symmetry_group('{self.symmetry_group}')`."
+        )
+        warnings.warn(msg, UserWarning, stacklevel=4)
+
+    def _result_symmetry(self, other=None):
+        """
+        Check if `other` will alter symmetry of `self` and return resulting symmetry_group.
+
+        :return: SymmetryGroup instance.
+        """
+        # No need to check for C1.
+        if isinstance(self.symmetry_group, IdentitySymmetryGroup):
+            return self.symmetry_group
+
+        # Warn and set to IdentitySymmetryGroup if `other` is symmetrically incompatible.
+        result_symmetry = self.symmetry_group
+
+        # Conditions of incompatibility.
+        self_transformation = other is None
```

Comment:
Why should self transformations always trigger reset to C1?

Reply:
This covers the cases of `rotate`, `transpose`, and `flip`; the "self-transformations" that do not maintain symmetry axes. The only self transformation that retains the symmetry group is `downsample`, which doesn't call `_result_symmetry` it just passes along `self.symmetry_group`.

Reply:
Wasn't this supposed to be renamed?

Reply:
Yep, sorry about that. I think I tagged you yesterday before the meeting. I was waiting for CI on another branch so I could update this. 

Changed the name to `axes_altering_transformation`